kansha/card/fts_schema.py

```diff
@@ -33,4 +34,5 @@ def from_model(cls, card):
                    comments='\n'.join(c.comment for c in card.comments),
                    lists='\n'.join(unicode(cl) for cl in card.checklists),
                    tags=' '.join(t.title for t in card.labels),
-                   board_id=card.column.board.id)
+                   board_id=card.column.board.id,
+                   archive=card.column.archive)
```

Comment:
A card can be archived, but is not an archive. So the propery would be more appropriately named "archived" or "in_archive" for clarity.